pkg/rbd/rbd_util.go

```diff
@@ -126,8 +127,27 @@ type rbdSnapshot struct {
 	RequestName    string
 }
 
+// ImageFeature represents required image features and value
+type ImageFeature struct {
+	// needRbdNbd indicates whether this image feature requires an rbd-nbd mounter
+	needRbdNbd bool
+	// required is the image features required for this ImageFeature
+	required []string
```

Comment:
This attribute name is a bit confusing IMO. Perhaps calling it `dependencies` or `dependsOn` would clarify what it means.

Reply:
yes, I will fix this. Thanks!